modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,101 @@
+
+. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.  
+See link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
++
+[cols="2,4,2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`https://cloud.redhat.com`
+|Used by the cluster for the `insights operator` that integrates with the SaaS Red Hat Insights
+|Required
+|===
+
+. For Amazon Web Services (AWS), you must grant access to the URLs that provide the AWS API and DNS:
++
+[cols="2,4,2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`*.amazonaws.com`
+|Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
+|Required
+|===
+
+. Allowlist the following URLs:
++
+[cols="2,4, 2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`mirror.openshift.com`
+|Required to access mirrored installation content and images. This site is also a source of release image signatures, although the Cluster Version Operator needs only a single functioning source.
+|Required
+
+|`storage.googleapis.com/openshift-release`
+|Alternative site to download platform release signatures that are used by the cluster to know what images to pull from quay.io.
+|Recommended
+
+|`*.apps.<cluster_name>.<shard>.<env>.openshiftapps.com:`
+|When allowlisting domains, use this in your corporate network to reach applications deployed in OpenShift, or to access the OpenShift console. The <shard> and <env> values are known after install time, and  <cluster_name> is set by the customer.
+|Required
+
+|`*.api.<cluster_name>.<shard>.<env>.openshiftapps.com:`
+|Used to access the public endpoint for the cluster API server. 
+|Required
+
+|`quay-registry.s3.amazonaws.com`
+|Used to access Quay image content in AWS.
+|Required
+
+|`api.openshift.com`
+|Required to check if updates are available for the cluster.
+|Required
+
+|`art-rhcos-ci.s3.amazonaws.com`
+|Specifies the {op-system-first} images to download.
+|Required
+
+|`cloud.redhat.com/openshift`
+|Required for your cluster token.
+|Required
+
+|`registry.access.redhat.com`
+| Access to the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
+|Required
+|===
++
+. Allowlist any site that provides resources for a language or framework that your builds require.
+. Allowlist any outbound URLs that depend on the languages and frameworks used in OpenShift. See link:https://access.redhat.com/solutions/2998411[OpenShift Outbound URLs to Allow] for a list of recommended URLs to be allowed on the firewall or proxy.
```

Comment:
We seem to be missing the domains needed for OSD Managed Services: https://docs.google.com/document/d/1xAyYnsi_8rwvZs3NqShUCKb1xvggR-HhUEAHCYD9Yqo/edit

Can you please add the following:
```
api.pagerduty.com, events.pagerduty.com: This alerting service is used by in-cluster alertmanager to send alerts to notifying SREs of an event to take action on.
Api.deadmanssnitch.com, nosnch.in: This alerting service is used by OSD to send periodic pings to indicate the cluster is available and running.
sftp.access.redhat.com: FTP server used by must-gather-operator to upload diagnostic logs to help troubleshoot issues with the cluster.
*.osdsecuritylogs.splunkcloud.com: Used by the splunk-forwarder-operator as a logging forwarding endpoint to be used by the SRE Platform Team for log-based alerting.
```

Reply:
These are added!

Reply:
Thanks, just noticed a slight typo with `Api.deadmanssnitch.com` that needs to be all lowercase `api.deadmanssnitch.com`